aviary/utils/fortran_to_aviary.py

```diff
@@ -663,6 +704,56 @@ def update_gasp_options(vehicle_data):
         ratios = [1 / val for val in ratios]
         input_values.set_val(Aircraft.Engine.Gearbox.GEAR_RATIO, ratios, units='unitless')
 
+    # CARGO
+    try:
+        if input_values.get_val(Aircraft.CrewPayload.Design.MAX_CARGO_MASS, 'lbm')[0] >= 0:
+            input_values.set_val(Aircraft.CrewPayload.Design.CARGO_MASS, [0.0], 'lbm')
+            input_values.set_val(Aircraft.CrewPayload.CARGO_MASS, [0.0], 'lbm')
+    except:
+        pass
+
+    # ENGINE
+    try:
+        num_engines = input_values.get_val(Aircraft.Engine.NUM_ENGINES, 'unitless')[0]
+        num_engines = int(num_engines)
+        input_values.set_val(Aircraft.Engine.NUM_ENGINES, [num_engines], 'unitless')
+        design_type = input_values.get_val(Aircraft.Design.TYPE, 'unitless')[0]
+        if design_type == 'BWB':
+            num_fuselage_engines = num_engines
+            # assume all engines are fuselage engines
+            input_values.set_val(
+                Aircraft.Engine.NUM_FUSELAGE_ENGINES, [num_fuselage_engines], 'unitless'
+            )
+            # BWB engine sizing algorithm does not use reference diameter
+            input_values.delete(Aircraft.Engine.REFERENCE_DIAMETER)
+    except:
+        pass
+    try:
+        engine_type = input_values.get_val(Aircraft.Engine.TYPE, 'unitless')[0]
+        if not (engine_type == 6 or engine_type == 7):
+            print('Only TURBOPROP(6) and TURBOJET(7) are allowed for now')
```

Comment:
```suggestion
            print(
                f'Engine type {engine_type} was provided; currently only TURBOPROP(6) and '
                'TURBOJET(7) are supported by Aviary'
            )
```
The formatting might be off here (hard to do in github), double check with ruff. Also, add a verbosity check before printing

Reply:
Done.